pkgs/applications/audio/r128gain/ffmpeg-location.patch

```diff
@@ -14,7 +12,7 @@ index 53fc3ef..f144e15 100755
 @@ -156,7 +156,7 @@ def get_r128_loudness(audio_filepaths, *, calc_peak=True, enable_ffmpeg_threadin
                                       os.devnull,
                                       **additional_ffmpeg_args,
-                                      f="null"),
+                                      f="null").global_args("-hide_banner", "-nostats"),
```

Comment:
Does the `-hide_banner` and `-nostats` belong here? Seems unrelated to the ffmpeg change, and if its not nixos-functionality related the patch probably doesn't belong here. Maybe its something to propose upstream?

Reply:
That's just the patch context, that line was changed upstream and I had to update it so the patch applied again.

Reply:
Oh I was confused by the diff of a diff. My bad.